tests/application-operator-tests/test/applicationcontroller/tests/applicationcontroller_test.go

```diff
@@ -54,3 +56,45 @@ func TestApplicationOperator_SkipProvisioning(t *testing.T) {
 
 	testSuite.CleanUp()
 }
+
+func TestApplicationOperator_LabelOverrides(t *testing.T) {
```

Comment:
Maybe it would be good idea to test negative case. For example if application is labeled with any other value (nochange.deployment.args.skipVerify) . This should not cause application upgrade and does not affect the  gateway deployment

Reply:
I do agree that such cases must be tested but I have put them in smaller unit tests not to make whatever runs against cluster even heavier. For example, the mentioned labels without overrides are tested by kyma/components/application-operator/pkg/kymahelm/application/overrides_test.go:11. I admit that the test case was not obvious. Let me just add another one there to assure that labels without override prefix does not cause any upgrades.